log/logClient.go

```diff
@@ -417,15 +428,24 @@ func WatchLogsHelper(res *pb.Log, o Options) error {
 
 		str = fmt.Sprintf("== Log / %s ==\n", updatedTime)
 
-		str = str + fmt.Sprintf("Cluster Name: %s\n", res.ClusterName)
-		str = str + fmt.Sprintf("Host Name: %s\n", res.HostName)
+		// Check if docker daemon is running
+		if _, err := os.Stat("/var/run/docker.pid"); err == nil {
+			dockerenabled = true // docker is running
+		}
```

Comment:
It is best to wrap this check in a function and just use that function. In the future, we will use more container runtimes and the internal check would differ.

Also it would be best to use some other name in place of `dockerenabled` so that we can use it for any container runtime.